### ISSUE TITLE
Typo in narrative docs for twisted.logger

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7875]
|[<img alt="derwolfe's avatar" src="https://avatars.githubusercontent.com/u/641357?s=50" width="50" height="50">](https://github.com/derwolfe)| @derwolfe reported|
|-|-|
|Trac ID|trac#7875|
|Type|defect|
|Created|2015-04-28 22:08:36Z|
|Branch|https://github.com/twisted/twisted/tree/logger-docs-typo-7875|

The narrative documentation for ``` twisted.logger ``` has a typo. 

Instead of

As such, it defaults to logging that the :api:`twisted.logger.LogLevel.critical <critical>` level.

the line should read:

As such, it defaults to logging at the :api:`twisted.logger.LogLevel.critical <critical>` level.

Attachments:

* [typo-logger-howto-docs-7875-1.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/255/255e71049836fcefee2f09418b210767d7061cca/957981de2961558ee9709087cf7703ee018a8fee.diff) (921 bytes) - added by herrwolfe on 2015-04-28 22:11:44Z - patch, newsfile
* [typo-logger-howto-docs-7875-2.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/255/255e71049836fcefee2f09418b210767d7061cca/8fee74a9b83f79d008f044ecc6058c75154996fc.diff) (919 bytes) - added by herrwolfe on 2015-04-28 22:19:21Z - fix ending on newsfile

<details><summary>Searchable metadata</summary>

```
trac-id__7875 7875
type__defect defect
reporter__herrwolfe herrwolfe
priority__normal normal
milestone__ 
branch__branches_logger_docs_typo_7875 branches/logger-docs-typo-7875
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1430258916314047 1430258916314047
changetime__1430261386826360 1430261386826360
version__None None
owner__herrwolfe herrwolfe

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="derwolfe's avatar" src="https://avatars.githubusercontent.com/u/641357?s=50" width="50" height="50">](https://github.com/derwolfe)|@derwolfe set owner to @derwolfe|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="derwolfe's avatar" src="https://avatars.githubusercontent.com/u/641357?s=50" width="50" height="50">](https://github.com/derwolfe)<a name="note_2"></a>|@derwolfe removed owner|
|-|-|

The patch https://twistedmatrix.com/trac/attachment/ticket/7875/typo-logger-howto-docs-7875-2.diff fixes the typo and adds an empty topfile. I left the topfile empty since the patch is a two character change.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

(In [44612]) Branching to logger-docs-typo-7875.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban set owner to @derwolfe|
|-|-|

changes looks good. thanks.... waiting for build to end before merging.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set status to `closed`|
|-|-|

(In [44614]) Merge logger-docs-typo-7875: Fix typo in narrative docs for twisted.logger.

Author: herrwolfe
Reviewer: adiroiban
Fixes: [#7875](https://github.com/twisted/twisted/issues/7875)
